std/range/package.d

```diff
@@ -7250,7 +7250,7 @@ if (isInputRange!Range)
     }
 
     // Assertion only.
-    private void dbgVerifySorted()
+    private auto dbgVerifySorted()
```

Comment:
You might be right, I don't know. I thought that they were only inferred if they were templates themselves.